mitodl/open-discussions
# Add membership field to Channel and REST API serializer

#### What are the relevant tickets?
Part of #864 

#### What's this PR do?
Adds a `membership_is_managed` field which defaults to true for channels created via REST API.

#### How should this be manually tested?
 - Create a channel in micromasters and verify that there is a `Channel` object in discussions for that channel and `membership_is_managed` is `True`
 - View `/api/v0/channels/` and verify that the field shows up in the results
